### DISCUSSION TITLE
GitHub Copilot could not connect to server. "Failed to get copilot token". Can not be fixed by several common methods.

### DISCUSSION THREAD

**Discussion by Physics-Lee** [community/community#76537] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

One day when I wanted to use Copilot as usual, I only got these (the output of GitHub Copilot Chat):

```powershell
2023-11-22T02:55:51.140Z [INFO] [auth] Logged in as Physics-Lee

2023-11-22T02:55:52.355Z [INFO] [auth] Invalid copilot token: missing token: 403 

2023-11-22T02:55:52.356Z [ERROR] [extension] GitHub Copilot could not connect to server. Extension activation failed: "Failed to get copilot token"
```

and these (the output of GitHub Copilot):

```powershell
[INFO] [auth] [2023-11-22T02:55:52.309Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-11-22T02:55:52.311Z] Extension activation failed: "Contact Support. You are currently logged in as Physics-Lee."
```

and these (the output of GitHub Copilot Labs):

```powershell
[ERROR] [default] [2023-11-22T02:55:51.573Z] GitHub Copilot Labs could not connect to server. Extension activation failed: "unable to verify the first certificate"
```

What I have tried and got failure:

* Try using diferent IDE, but VS code, VS, PyCharm all show the same mistake.
* Try to log out and log in back, but the same mistake is showed again.
* Try to uninstall and install the GitHub Copilot extension again, but the bug is showed again.
* Try to deactivate the subscription of GitHub Copilot and activate it again, but the the same mistake still exists.

**Comment by Physics-Lee** [comment]
My Copilot:
![1](https://github.com/community/community/assets/68525696/5ee26d9c-34b9-47a5-91cf-88f9b318a968)


My Billing Plan:
![2](https://github.com/community/community/assets/68525696/2b7397aa-521a-4f8e-a918-3d78078cc83a)


**Is there anything wrong with the first figure? Do you also show these 2 figs?**

**Reply by bodhianhaha** [reply]
me too ! How to fix this problem?

**Reply by Physics-Lee** [reply]
I figured it out by the advise from @Akash1134 ~

**Comment by Akash1134** [comment, marked as answer]
Hi @Physics-Lee , This seems like an issue with your account. The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by Physics-Lee** [reply]
Thanks~
